### PR TITLE
rename kubernetes module to kubernetesmod

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
This makes sure this doesn't import the kubernetes module from
salt/modules while the kubernetes python module is not installed

### What issues does this PR fix or reference?
Issue brought up in community slack

### Tests written?

Yes

### Commits signed with GPG?

Yes

### CODE REVIEW

**Review by cachedout — APPROVED** [review]
LGTM

File: salt/states/kubernetes.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ==========================================
```
Comment (dwoz) [inline salt/states/kubernetes.py]: Could this have been solved using `absolute_imports` here?

Comment (gtmanfred) [reply salt/states/kubernetes.py]: No, because we add salt/modules to sys.path in the loader.

Comment (gtmanfred) [reply salt/states/kubernetes.py]: https://github.com/saltstack/salt/blob/2018.3/salt/loader.py#L1453

**Pull request merged**